### PR TITLE
Remove http.status_text from span attributes

### PR DESCRIPTION
**Pull request by lonewolf3739**
# Description

`http.status_text` was removed in this spec pr https://github.com/open-telemetry/opentelemetry-specification/pull/972

### CODE REVIEW

File: instrumentation/opentelemetry-instrumentation-urllib/src/opentelemetry/instrumentation/urllib/__init__.py [hunk 1]
```diff
@@ -171,7 +171,6 @@ def _instrumented_open_call(
 
                 if span.is_recording():
                     span.set_attribute("http.status_code", code_)
-                    span.set_attribute("http.status_text", result.reason)
                     span.set_status(Status(http_status_to_status_code(code_)))
```
Comment (codeboten) [inline instrumentation/opentelemetry-instrumentation-urllib/src/opentelemetry/instrumentation/urllib/__init__.py:174]: should the reason be set on the status?

File: instrumentation/opentelemetry-instrumentation-urllib3/src/opentelemetry/instrumentation/urllib3/__init__.py [hunk 2]
```diff
@@ -203,7 +203,6 @@ def _apply_response(span: Span, response: urllib3.response.HTTPResponse):
         return
 
     span.set_attribute("http.status_code", response.status)
-    span.set_attribute("http.status_text", response.reason)
     span.set_status(Status(http_status_to_status_code(response.status)))
```
Comment (codeboten) [inline instrumentation/opentelemetry-instrumentation-urllib3/src/opentelemetry/instrumentation/urllib3/__init__.py:206]: same question about setting the reason on the status here

File: instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py [hunk 3]
```diff
@@ -157,8 +157,7 @@ def add_response_attributes(
     passed to a PEP3333-conforming start_response callable."""
     if not span.is_recording():
         return
-    status_code, status_text = start_response_status.split(" ", 1)
```
Comment (codeboten) [inline instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py:160]: same question about the status_text being set on the status

**Review by codeboten — COMMENTED** [review]
Thanks for taking on this change, I noticed some of the instrumentation uses the text from status_text in the status description, we should be consistent one way or the other about it.

File: instrumentation/opentelemetry-instrumentation-urllib/src/opentelemetry/instrumentation/urllib/__init__.py (hunk 1 above)
Comment (lonewolf3739) [reply instrumentation/opentelemetry-instrumentation-urllib/src/opentelemetry/instrumentation/urllib/__init__.py:174]: Personally I don't think setting a reason phrase adds any value when there is already status code. Copying the same from [RFC](https://tools.ietf.org/html/rfc7230#section-3.1.2). 
> The reason-phrase element exists for the sole purpose of providing a
   textual description associated with the numeric status code, mostly
   out of deference to earlier Internet application protocols that were
   more frequently used with interactive text clients.  A client SHOULD
   ignore the reason-phrase content.

Some places reason is not really http reason phrase it is just formatted exception. I would suggest removing reason phrase on status, WDYT?

Comment (codeboten) [reply instrumentation/opentelemetry-instrumentation-urllib/src/opentelemetry/instrumentation/urllib/__init__.py:174]: Ah i see, for some reason I was thinking that `reason` would be more than the status message for the http response

**Review by codeboten — APPROVED** [review]

**Pull request merged**